element_electrode_localization/coordinate_framework.py

```diff
@@ -105,24 +154,14 @@ class ParentBrainRegion(dj.Lookup):
 def load_ccf_annotation(
     ccf_id, version_name, voxel_resolution, nrrd_filepath, ontology_csv_filepath
 ):
-    """
-    :param ccf_id: unique id to identify a new CCF dataset to be inserted
-    :param version_name: CCF version
-    :param voxel_resolution: voxel resolution in micron
-    :param nrrd_filepath: path to the .nrrd file for the volume data
-    :param ontology_csv_filepath: path to the .csv file for the brain region ontology
-
-    load_ccf_annotation(
-        ccf_id=0, version_name='ccf_2017', voxel_resolution=10,
-        nrrd_filepath='./data/annotation_10.nrrd',
-        ontology_csv_filepath='./data/query.csv')
-
-    For an example Allen brain atlas for mouse, see:
-    http://download.alleninstitute.org/informatics-archive/current-release/mouse_ccf/annotation/ccf_2017
-
-    For the structure/ontology tree, see:
-    https://community.brain-map.org/t/allen-mouse-ccf-accessing-and-using-related-data-and-tools/359
-    (particularly the ontology file downloadable as CSV)
+    """Load CCF annotation
+
```

Comment:
Google style allows a more substantive description after the oneliner, and then an Example section. Please add back these pieces.

Reply:
Added